examples/exporting_for_inference.ipynb

```diff
@@ -7,22 +7,7 @@
    "source": [
     "# 🥡 Exporting for Inference\n",
     "\n",
-    "Composer models are also `torch.nn.Module`, and thus can be exported like any other PyTorch module. In this tutorial, we walk through how to export your models into various common formats: [ONNX](https://onnx.ai/), [TorchScript](https://pytorch.org/docs/stable/jit.html), and [torch.fx](https://pytorch.org/docs/stable/fx.html). For more detailed options and configuration settings, please consult the linked documentation.\n",
-    "\n",
-    "## Algorithm compatibility\n",
-    "Some of our algorithms alter the model architecture in ways that may render them incompatible with some of the export procedures above. For example, BlurPool replaces some instances of `Conv2d` with `BlurConv2d` layers which are not compatible with `torch.fx` as they have data-dependant control flow. \n",
-    "\n",
-    "The following table shows which algorithms are compatible with which export formats for inference.\n",
-    "\n",
-    "|                        | torchscript | torch.fx | ONNX |\n",
-    "|------------------------|-------------|----------|------|\n",
-    "| apply_blurpool         | &check;           |          | &check;    |\n",
-    "| apply_factorization    |             | &check;        | &check;    |\n",
-    "| apply_ghost_batchnorm  | &check;           |          | &check;    |\n",
-    "| apply_squeeze_excite   | &check;           | &check;        | &check;    |\n",
-    "| apply_stochastic_depth | &check;           | &check;        | &check;    |\n",
-    "| apply_channels_last    | &check;           | &check;        | &check;    |\n",
-    "\n"
+    "Composer provides model export support for inference using a callback and also functional API.  In this tutorial, we walk through how to export your models into various common formats: [ONNX](https://onnx.ai/), [TorchScript](https://pytorch.org/docs/stable/jit.html) using Composer's callback mechanism as well as functional API. For more detailed options and configuration settings, please consult the linked documentation. In addition, if for any reason, above methods of exporting are not sufficient for your use case Composer models can be exported like any other PyTorch module since Composer models are also `torch.nn.Module`. "
```

Comment:
functional API is a bit confusing, as have the algorithms functional API.

Reply:
would calling it standalone API help?

Reply:
I call it dedicated export API now.